internal/testdata/snapshots/output/alias/main.go

```diff
@@ -47,5 +61,23 @@
 // > struct{}
 // > ```
 //   ^ reference github.com/sourcegraph/scip-go . `github.com/sourcegraph/scip-go/internal/testdata/snapshots/input/alias`/U#
+   Z int32
+// ^ definition github.com/sourcegraph/scip-go . `github.com/sourcegraph/scip-go/internal/testdata/snapshots/input/alias`/Z#
+// documentation
+// > Check that we don't panic
+// > Copied from https://github.com/golang/go/issues/68877#issuecomment-2290000187
+// documentation
+// > ```go
+// > int32
```

Comment:
This is the bug in https://github.com/sourcegraph/scip-go/issues/107 -- it's bizarre that this was not handled when the original switch was written, but trying to not fix that here for now, will do that in a follow-up PR